src/LanguageServer/LanguageServer.csproj

```diff
@@ -15,7 +15,6 @@
     <PackageReference Include="OmniSharp.Extensions.LanguageProtocol" Version="0.7.9" />
     <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="0.7.9" />
     <PackageReference Include="Microsoft.Build" Version="16.7.0" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.Build.Locator" Version="1.2.6" />
```

Comment:
This project doesn't use `MSBuildLocator` directly, so it's better to remove an explicit dependency